---
title: "Welcome to the Waldron Lab Shiny Server!"
author:
  - name: Waldron Lab
    affiliation: |
      CUNY School of Public Health and Health Policy &
      CUNY Institute for Implementation Science in Population Health
date: "`r BiocStyle::doc_date()`"
output:
  BiocStyle::html_document:
    self_contained: yes
    toc: no
---

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  echo = FALSE,
  include = FALSE
)
```

```{r, include=FALSE}
okapps <- c(
    "/srv/shiny-server/BiocHubsShiny",
    "/srv/shiny-server/BugSigDBEnrich",
    "/srv/shiny-server/curatedMetagenomicDataCurationShiny",
    "/srv/shiny-server/PowerCalc",
    "/srv/shiny-server/primers"
)
applinks <- paste("https://shiny.sph.cuny.edu", basename(okapps), sep = "/")
picfiles <- file.path("~/pictures", paste0(basename(okapps), ".png"))
stopifnot(
    all(file.exists(picfiles))
)
```

# BiocHubsShiny

```{r,out.width='100%', fig.link = applinks[1], include=TRUE, echo=FALSE}
knitr::include_graphics(picfiles[1])
```

# BugSigDBEnrich

```{r,out.width='100%', fig.link = applinks[2], include=TRUE, echo=FALSE}
knitr::include_graphics(picfiles[2])
```

# curatedMetagenomicDataCurationShiny

```{r,out.width='100%', fig.link = applinks[3], include=TRUE, echo=FALSE}
knitr::include_graphics(picfiles[3])
```

# PowerCalc

```{r,out.width='100%', fig.link = applinks[4], include=TRUE, echo=FALSE}
knitr::include_graphics(picfiles[4])
```

# R Primers

```{r,out.width='100%', fig.link = applinks[5], include=TRUE, echo=FALSE}
knitr::include_graphics(picfiles[5])
```
